live/tests/integration/components/qcm-solution-panel-test.js

```diff
@@ -0,0 +1,215 @@
+import {expect} from 'chai';
+import {describe, it} from 'mocha';
+import {setupComponentTest} from 'ember-mocha';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+import _ from 'pix-live/utils/lodash-custom';
+
+const CHECKBOX_CORRECT_AND_CHECKED = '.qcm-panel__proposal-checkbox:eq(1)';
+const LABEL_CORRECT_AND_CHECKED = '.qcm-proposal-label__oracle:eq(1)';
+
+const CHECKBOX_CORRECT_AND_UNCHECKED = '.qcm-panel__proposal-checkbox:eq(2)';
+const LABEL_CORRECT_AND_UNCHECKED = '.qcm-proposal-label__oracle:eq(2)';
+
+const CHECKBOX_INCORRECT_AND_CHECKED = '.qcm-panel__proposal-checkbox:eq(3)';
+const LABEL_INCORRECT_AND_CHECKED = '.qcm-proposal-label__oracle:eq(3)';
+
+const CHECKBOX_INCORRECT_AND_UNCHECKED = '.qcm-panel__proposal-checkbox:eq(0)';
+const LABEL_INCORRECT_AND_UNCHECKED = '.qcm-proposal-label__oracle:eq(0)';
+
+
+const CSS_BOLD_FONT_WEIGHT = '900';
+const CSS_NORMAL_FONT_WEIGHT = '400';
+
+const CSS_GREEN_COLOR = 'rgb(19, 201, 160)';
+const CSS_BLACK_COLOR = 'rgb(51, 51, 51)';
+
+const CSS_LINETHROUGH_ON = 'line-through';
+const CSS_LINETHROUGH_OFF = 'none';
+
+function charCount(str) {
+  return str.match(/[a-zA-Z]/g).length;
+}
+
+describe('Integration | Component | qcm-solution-panel.js', function () {
+  setupComponentTest('qcm-solution-panel', {
+    integration: true
+  });
+
+  describe('#Component should renders: ', function () {
+
+    it('Should renders', function () {
+      this.render(hbs`{{qcm-solution-panel}}`);
+      expect(this.$()).to.have.length(1);
+      expect($(LABEL_CORRECT_AND_CHECKED)).to.have.lengthOf(0);
+    });
+
+    describe('checkbox state', function () {
+
```

Comment:
Les lignes 51 à 67 se répètent dans le fichier si je ne me trompe pas, il faudrait les sortir des it et les mettre juste en-dessous du describe.